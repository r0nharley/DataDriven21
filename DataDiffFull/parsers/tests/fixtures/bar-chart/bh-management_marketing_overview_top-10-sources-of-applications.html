<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="699" height="343">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-ir61qbo-338">
            <rect x="0" y="0" width="297" height="542" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-ir61qbo-339">
            <rect x="0" y="0" width="297" height="542" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="699" height="343" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="147" y="10" width="542" height="297"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 147 40.5 L 689 40.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 69.5 L 689 69.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 99.5 L 689 99.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 129.5 L 689 129.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 159.5 L 689 159.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 188.5 L 689 188.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 218.5 L 689 218.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 248.5 L 689 248.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 277.5 L 689 277.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 307.5 L 689 307.5" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 147 10.5 L 689 10.5" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 146.5 10 L 146.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 200.5 10 L 200.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 254.5 10 L 254.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 309.5 10 L 309.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 363.5 10 L 363.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 417.5 10 L 417.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 471.5 10 L 471.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 525.5 10 L 525.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 580.5 10 L 580.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 634.5 10 L 634.5 307" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 689.5 10 L 689.5 307" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="147" y="10" width="542" height="297"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 40.5 L 137 40.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 69.5 L 137 69.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 99.5 L 137 99.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 129.5 L 137 129.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 159.5 L 137 159.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 188.5 L 137 188.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 218.5 L 137 218.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 248.5 L 137 248.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 277.5 L 137 277.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 307.5 L 137 307.5" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 147 9.5 L 137 9.5" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 146.5 10 L 146.5 307"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 147 307 L 689 307"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker " transform="translate(689,307) rotate(90) scale(-1,1) scale(1 1)" width="542" height="297" clip-path="url(http://local.dev/bh-management/bi/marketing-overview/visualization#highcharts-ir61qbo-339)">
            <rect x="274.5" y="12.5" width="15" height="530" fill="#00cee6" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="244.5" y="312.5" width="15" height="230" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="215.5" y="347.5" width="15" height="195" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="185.5" y="432.5" width="15" height="110" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="155.5" y="472.5" width="15" height="70" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="126.5" y="477.5" width="15" height="65" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="96.5" y="478.5" width="15" height="64" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="66.5" y="480.5" width="15" height="62" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="36.5" y="482.5" width="15" height="60" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="7.5" y="487.5" width="15" height="55" fill="#00cee6" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-bar-series highcharts-color-undefined " transform="translate(689,307) rotate(90) scale(-1,1) scale(1 1)" width="542" height="297" clip-path="none"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-bar-series highcharts-color-undefined highcharts-tracker" transform="translate(147,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(485,3)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(230,33)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(195,62)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(110,92)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(70,122)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(65,151)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(64,181)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(62,211)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(60,241)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-undefined highcharts-tracker" style="" transform="translate(55,270)"></g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="29" opacity="1">
            <tspan>Property Website</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="59" opacity="1">
            <tspan>portal</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="89" opacity="1">
            <tspan>Apartments.com</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="118" opacity="1">
            <tspan>Resident Referral</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="148" opacity="1">
            <tspan>RENTCafe.com ILS</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="178" opacity="1">
            <tspan>Walk in/Drive By</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="208" opacity="1">
            <tspan>BH Website</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="237" opacity="1">
            <tspan>Walk-in/Drive-by</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="267" opacity="1">
            <tspan>Drive By</tspan>
        </text>
        <text x="132" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:221px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="297" opacity="1">
            <tspan>ApartmentGuide.com</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="147" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">0</text>
        <text x="201.2" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">1K</text>
        <text x="255.39999999999998" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">2K</text>
        <text x="309.6" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">3K</text>
        <text x="363.79999999999995" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">4K</text>
        <text x="418" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">5K</text>
        <text x="472.2" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">6K</text>
        <text x="526.4" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">7K</text>
        <text x="580.5999999999999" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">8K</text>
        <text x="634.8" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">9K</text>
        <text x="677.9609375" style="color:#666666;cursor:default;font-size:11px;fill:#666666;" text-anchor="middle" transform="translate(0,0)" y="326" opacity="1">10K</text>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(9,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 132 0 C 132 0 132 0 132 0 L 132 17 138 23 132 29 132 102 C 132 102 132 102 132 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>