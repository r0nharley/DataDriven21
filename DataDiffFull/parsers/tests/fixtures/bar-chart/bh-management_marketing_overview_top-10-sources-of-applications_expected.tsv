Property Website	Data	9778.59778598
portal	Data	4243.54243542
Apartments.com	Data	3597.78597786
Resident Referral	Data	2029.5202952
RENTCafe.com ILS	Data	1291.51291513
Walk in/Drive By	Data	1199.26199262
BH Website	Data	1180.81180812
Walk-in/Drive-by	Data	1143.91143911
Drive By	Data	1107.01107011
ApartmentGuide.com	Data	1014.7601476